subprojects/parseq-benchmark/build.gradle

```diff
@@ -16,7 +16,6 @@ task fatJar(type: Jar) {
   manifest {
     attributes("Created-By": "Gradle",
         "Version": version,
-        "Built-By": project.rootProject.ext.builtByUser,
```

Comment:
Curious about why do we remove this one?

Reply:
According to Szczepan, this sort of "volatile" metadata can harm cacheability. See: https://github.com/linkedin/parseq/pull/243#discussion_r404343718

Also, since all the publications will be done from Travis using a certain person's Bintray credentials, it doesn't seem appropriate.

Reply:
Good to know that.